src/librustc_resolve/lib.rs

```diff
@@ -961,6 +961,8 @@ pub struct Resolver<'a> {
     /// Stores enum visibilities to properly build a reduced graph
     /// when visiting the correspondent variants.
     variant_vis: DefIdMap<ty::Visibility>,
+
+    pub lint_buffer: lint::LintBuffer,
```

Comment:
Could you perhaps make a getter instead of `pub`? All other resolver fields are private.

Reply:
I think this changed while I was working on this, but I've now done so.